src/mqt/bench/cli.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+
+import argparse
+
+from pytket import Circuit
+from pytket.qasm import circuit_to_qasm_str
+from qiskit import QuantumCircuit
+from qiskit.qasm2 import dumps as qiskit_circuit_to_str
+
+from mqt.bench import CompilerSettings, QiskitSettings, TKETSettings, get_benchmark
+
+
+def main() -> None:
+    """
+    Invoke :func:`get_benchmark` exactly once with the specified arguments.
+
+    The resulting QASM string is printed to the standard output stream;
+    no other output is generated.
+    """
+
+    parser = argparse.ArgumentParser(description="Generate a single benchmark")
+    parser.add_argument(
+        "--level",
+        type=str,
+        help='Level to generate benchmarks for ("alg", "indep", "nativegates" or "mapped")',
+        required=True,
+    )
+    parser.add_argument("--algorithm", type=str, help="Name of the benchmark", required=True)
```

Comment:
I'd guess that this argument should also be required. Doesn't really make sense to generate a benchmark without knowing which algorithm to generate.

Reply:
It is `required` though, isn't it? :D

Reply:
Yap. Nevermind. Must have overlooked that while reviewing on the phone 🙃